released-by-github/query.sql

```diff
@@ -0,0 +1,29 @@
, "kzadeh", "lachlanhardy", "lee-dohm", "lilliSeBe", "LizzHale", "LizzyKemp", "maki1022", "marcusramos", "marisa", "megbird", "mellybess", "mfrosen", "micahcroff", "michelemt", "michellini", "MikeNwin", "missinformed", "mlinksva", "Mlohoury", "msbise", "mwiesen", "NadaAldahleh", "NataliaHarakka", "nelson-steven", "nickcannariato", "nixpad", "nplasterer", "nsqe", "ohitsmekatie", "omgitsads", "paladique", "papimenon", "parkr", "parthd24", "patriciadavico", "pholleran", "pmarsceill", "pnsk", "rafer", "remywestin", "rewinfrey", "Richardstraat", "rkarthikeyan5", "rohinisf", "ronnie", "samuelhunt", "sandhub", "saraford", "sherrinetan", "shlomi-noach", "shortsparty", "Shreekar1", "simurai", "sjarolimek", "sleepysam", "snh", "sophshep", "staceb", "stevecat", "stoe", "suniljoseph", "suziejurado", "svenstaa", "swinton", "tarebyte", "tberman", "theseawitch", "tiabobia", "ticachica", "tierninho", "tma", "tmaher", "tomkrouper", "toyae", "veesa", "vinmat", "vmramirez", "webdog", "WillAbides", "wingr", "woodmanzee", "xducati620", "xhermann", "yourfrienderin", "zawaideh", "dekigai", "pifafu", "reversaw", "tcbyrd", "Teaford", "tenderlove", "kganes2", "neev7", "lecoursen", "ThatStoney", "brandonblack", "CoralineAda", "affrae", "AlexAmsterdam", "AlexPurkiss", "aymannadeem", "bluemazzoo", "carriemoser", "dmleong", "dolcetta", "KT2016", "leithiaRVA", "miguelff", "ntagg", "rhettg", "sambong", "sarabartell", "shreyasjoshis", "skimbro", "sofiehoeg", "tmaltiel", "tstone92", "victoiremulliez", "brianamarie", "fossygirl", "kamilahgriffin", "katmeister", "mistydemeo", "peggysf", "rosecoogan", "sbarnekow", "zeke", "AndrewCamp", "doublemarket", "bungdonuts", "chronicleofnerdia", "halfaro1", "harrisonleavitt", "hktouw", "jkrmr", "katfoz", "kdraps", "acmccall", "kneemer", "KristinaMarie", "mclark", "NatalieGrace", "arthurschreiber", "chobberoni", "craigmorrall", "osowskit", "el33", "dimmykumont", "fabien965", "jamfish", "jonico", "k33g", "m-air", "MarcoHat", "mosborne24", "nadiajoyce", "othercasey", "pravipati", "reem-a", "RuedigerEbber", "SWAKA", "wordsoup", "EprisBlank", "timothywellsjr", "chadjlucas", "hugginz", "mikrobi", "AllisonZhornist", "cvfernandez", "grokys", "jialeec", "kivikakk", "Lieke22", "mfrisky", "rubyjazzy", "seemakamath", "Tracol", "evilshawn", "NikiLustig", "panand71", "as-cii", "Raulv81", "rushtonmd", "kytrinyx", "lowply", "mcadoo", "Nickismyname", "terrorobe", "freakingid", "ACyphus", "amitbhatia", "beardofedu", "bevns", "kemjo", "mslilylove", "pbdunn", "sridharavinash", "groundwater", "stephaniemaria", "hollenberry", "JodiDelman", "melscoop", "NedMorris", "ttaylorr", "brentjo", "antn", "BastiaanHendriks", "Briroth", "bryancross", "canuckjacq", "doctorwholock", "femisimon", "heyhayhay", "hubberjay", "jules-p", "kathodos", "ktfeucht", "lukeart", "oriandaleigh", "shawnajean", "liliawu", "piqrgb", "camicano", "corywilkerson", "damieng", "emrichard", "nayafia", "ryanpamela", "sophaskins", "skottler", "swannysec", "joenash", "migarjo", "talktopri", "wi11ialvl")
```

Comment:
Gah, unfortunately the `PublicEvent` uses the name of the organization and not the real actor.